plugins/scaffolder-react/src/next/components/Workflow/Workflow.test.tsx

```diff
@@ -142,6 +142,8 @@ describe('<Workflow />', () => {
       fireEvent.click(getAllByRole('button')[1] as HTMLButtonElement);
     });
 
+    await new Promise(resolve => setTimeout(resolve, 250));
```

Comment:
never use fixed waits, wrap the expectation below in a `waitFor` instead (see elsewhere in the repo for examples). The same applies for all of the other test files in the PR

Reply:
<img width="700" alt="image" src="https://github.com/user-attachments/assets/f40b4955-1ad3-4dcb-a567-5bc7c62bad9e" />

This pattern is copied from other test files. And before `create` fired, no DOM element will be changed? I don't know what's the best alternative, maybe `jest. useFakeTimers()`?